client/src/AppBundle/Resources/views/Report/Report/overview.html.twig

```diff
@@ -315,7 +315,7 @@
     {# Fallback for JS disabled browsers #}
     <div class="js-hidden">
         <a id="{{ report.status.isReadyToSubmit ? 'edit-report-review' : 'edit-report-preview' }}"
-           class="push--top behat-link-edit-report-review"
```

Comment:
One of the issues I was noticing was that when behat searching for the `behat-link-edit-report-review` class, it would fine two cases of it on this page and would complain and end the tests. I believe this was because it wouldn't be able to determine which link to click.

I removed it from here because this was for fallback, and it should only affect the behat tests. Still need to confirm if this breaks any tests once the pipeline issues are fixed.